pypesto/result/sample.py

```diff
@@ -100,19 +184,16 @@ def __init__(
                 f"trace_neglogprior.shape={trace_neglogprior.shape}"
             )  # noqa
 
-    def __getattr__(self, key):
-        """Allow usage of keys like attributes."""
-        try:
-            return self[key]
-        except KeyError:
-            raise AttributeError(key)
-
     __setattr__ = dict.__setitem__
     __delattr__ = dict.__delitem__
 
-
-class SampleResult:
-    """Result of the sample() function."""
-
-    def __init__(self):
-        pass
+    def __next__(self):
```

Comment:
this looks like a duplicate

Reply:
Nope, as you have another indexing due to different chains... :/